CHANGES.rst

```diff
@@ -113,6 +113,13 @@ master_background
 - Updated documentation to more fully describe the various ways in which the
   step is applied [#5913]
 
+outlier_detection
+-----------------
+
+- outlier detection on non-dithered images is implemented with a simple sigma clipping
+- dithered outlier detection cleaned up and HST specific steps removed
+- additional tests added
+
```

Comment:
Please add a reference to the PR number, i.e. [#5822], at the end of each of these lines (or consolidate them into a single entry). See other entries for examples.

Reply:
Done.